gcloud/datastore/key.py

```diff
@@ -167,10 +169,10 @@ def completed_key(self, id_or_name):
         """Creates new key from existing partial key by adding final ID/name.
 
         :rtype: :class:`gcloud.datastore.key.Key`
-        :returns: A new `Key` instance with the same data as the current one
+        :returns: A new ``Key`` instance with the same data as the current one
                   and an extra ID or name added.
-        :raises: `ValueError` if the current key is not partial or if
-                 `id_or_name` is not a string or integer.
+        :raises: :class:`ValueError` if the current key is not partial or if
+                 ``id_or_name`` is not a string or integer.
```

Comment:
Will add blocks. 

Any user is allowed to complete a key, not just the backend via `auto_id`, so a name is just fine here.